lms/djangoapps/student_account/views.py

```diff
@@ -344,26 +343,27 @@ def get_user_orders(user):
 
     for order in commerce_user_orders:
         if order['status'].lower() == 'complete':
-            for line in order['lines']:
-                product = line.get('product')
-                if product:
-                    for attribute in product['attribute_values']:
-                        if attribute['name'] == 'certificate_type' and attribute['value'] in allowed_course_modes:
-                            try:
-                                date_placed = datetime.strptime(order['date_placed'], "%Y-%m-%dT%H:%M:%SZ")
-                                order_data = {
-                                    'number': order['number'],
-                                    'price': order['total_excl_tax'],
-                                    'title': order['lines'][0]['title'],
-                                    'order_date': strftime_localized(
-                                        date_placed.replace(tzinfo=pytz.UTC), 'SHORT_DATE'
-                                    ),
-                                    'receipt_url': EcommerceService().get_receipt_page_url(order['number'])
-                                }
-                                user_orders.append(order_data)
-                            except KeyError:
-                                log.exception('Invalid order structure: %r', order)
-                                return no_data
+            date_placed = datetime.strptime(order['date_placed'], "%Y-%m-%dT%H:%M:%SZ")
+            order_data = {
+                'number': order['number'],
+                'price': order['total_excl_tax'],
+                'order_date': strftime_localized(
+                    date_placed.replace(tzinfo=pytz.UTC), 'SHORT_DATE'
```

Comment:
The code was already there. I assumed it had some purpose and didn't think about it. Removed it and it works fine without that part.